eusurdf/euslisp/eusmodel_to_urdf.l

```diff
@@ -0,0 +1,59 @@
+;; eusmodel_to_urdf.l
+;; Author: Yuki Furuta <furushchev@jsk.imi.i.u-tokyo.ac.jp>
+
+;; assumed executed at PROJECT_SOURCE_DIR
+(defvar *eusurdf-package-dir* (unix:getwd))
+(require :conversion-utils (format nil "~A/euslisp/conversion-utils.l" *eusurdf-package-dir*))
+
+(defun make-model-conf (name fpath &optional desc)
+  (unless desc
+    (setq desc (format nil "This is automatically generated eus model from ~A" name)))
+  (with-open-file (f fpath :direction :output :if-exists :overwrite)
+    (format f "<?xml version='1.0'?>~%")
+    (format f "<model>~%")
+    (format f "  <name>~A</name>~%" name)
+    (format f "  <version>0.1.0</version>~%")
+    (format f "  <sdf>model.urdf</sdf>~%")
+    (format f "  <description>~%")
+    (format f "    ~A~%" desc)
+    (format f "  </description>~%")
+    (format f "</model>~%")))
+
+(defun eusmodelfile2urdf (fname outdir &key (tmpdir "/tmp/"))
+  (unless (probe-file fname)
+    (errorf "file ~A not exists" fname))
+  (load fname)
+  (let* ((model-name (string-join "-" (butlast (string-split (send (pathname fname) :name) #\-))))
+         (model (funcall (read-from-string model-name))))
+    (eusmodel2urdf model model-name outdir :tmpdir tmpdir)))
+
+(defun eusmodel2urdf (model model-name outdir &key (tmpdir "/tmp/") collada-to-urdf-exe-path)
+  (let ((urdf-out-path (send (concatenate-pathnames outdir "model.urdf") :namestring))
+        (mesh-out-dir (send (concatenate-pathnames outdir "meshes/") :namestring)))
+    (cond ((null (send model :name)) (send model :name model-name))
+          ((symbolp (send model :name)) (send model :name (string-downcase (send model :name)))))
+    (collada::eus2collada model tmpdir)
+    (make-dirs mesh-out-dir)
+    (unless collada-to-urdf-exe-path
+      (setq collada-to-urdf-exe-path "rosrun collada_urdf_jsk_patch collada_to_urdf"))
```

Comment:
https://github.com/jsk-ros-pkg/jsk_model_tools/pull/160#issuecomment-162804661 should pass for passing deb build. and rosrun cannot be used (at least at this Issue's timing)